decompiler/util/serialization/bytes_serializer.py

```diff
@@ -7,7 +7,7 @@
 def convert_bytes(bytestring: bytes, vartype: Type, endian: int = 0) -> int:
     """Serialize a bytestring into a number representation if possible"""
     if isinstance(vartype, Pointer):
-        if vartype.type.size == 8:
+        if vartype.type.size == 8 or vartype.type.size == 0:
```

Comment:
What is happening here? Do you check for void types?

Reply:
Yes correct. Without the check void pointer will be converted to a numerical value.

For example: Without check
```C
extern void * data_2004 = 25637;
```
With check
```C
extern void * data_2004 = "\x25\x64\x00\x00";
```

Reply:
Kinda hate the hardcoded 8 here, but your addition seems fine.

Reply:
:D Also done then